### DISCUSSION TITLE
Removing VS Ignoring a Package

### DISCUSSION THREAD

**Discussion by Bad3r** [CyberShadow/aconfmgr#108] [Q&A] [answered]
Hi,
I am somewhat confused about which command should I use. On my current system, there are some packages that I don't want to be tracked or installed on my 2nd computer. In this case, should I ignore the packages or use the remove command in the config? Do I need to use the remove command first and remove the occurrences of the packages, then add the ignore command?

**Comment by CyberShadow** [comment, marked as answer]
The documentation for `RemovePackage` [states](https://github.com/CyberShadow/aconfmgr/blob/c1b1d06db1097ac8282a0297d5b464afcef7eb27/src/helpers.bash#L35-L42):

> Removes an earlier-added package to the list of packages to be installed.
>
> Emitted by `aconfmgr save` when a package is present in the configuration, but absent on the system.
>
> You should refactor out any occurrences of this function from your configuration.

So, `RemovePackage` simply undoes the effect of an `AddPackage` line earlier in the configuration.

> On my current system, there are some packages that I don't want to be tracked or installed on my 2nd computer.

`IgnorePackage` would work for this purpose. However, consider instead if your goal would be better met by making the `AddPackage` lines for these packages conditional. See [the README section "Managing multiple systems"](https://github.com/CyberShadow/aconfmgr#managing-multiple-systems) for more information and examples.
